apps/application-system/api/src/app/modules/application/tools/applicationAccess.service.ts

```diff
@@ -49,16 +55,36 @@ export class ApplicationAccessService {
     return existingApplication as BaseApplication
   }
 
-  async canDeleteApplication(
+  async isUserActorOnApplication(id: string, user: User) {
```

Comment:
I would expect a function named isUserActorOnApplication to return boolean true if the user is an actor on the application. Or atleast I dont quite get why it throws a bad subject if the user is an actor.

Reply:
Yeah, that makes sense. I'll move the code back, I was aiming for code readability but it's maybe just better kept intact.